src/Template/rector.php

```diff
@@ -61,6 +61,8 @@
         realpath(getcwd()) . '/vendor/codeigniter4/framework/system/Test/bootstrap.php',
     ]);
 
+    $parameters->set(Option::PHPSTAN_FOR_RECTOR_PATH, __DIR__ . '/phpstan.neon.dist');
```

Comment:
probably needs `file_exists(__DIR__ . '/phpstan.neon.dist')` check first?

Reply:
My assumption is that all the files in **Template/** can rely on each other, and modifications made by users (like picking and choosing files) will already need adjustment. In this case I guess that's a pretty minor change so not a bad idea.